### ISSUE TITLE
Add brand for Masonic Temple

### ISSUE THREAD

**Issue by quincylvania** [osmlab/name-suggestion-index#2846]
Similar to #2845. These are also tagged inconsistently.

https://en.wikipedia.org/wiki/Masonic_Temple

**Comment by 1ec5** [comment]
Masonic Temples are sort of getting into a gray area… from what I’ve seen, the level of branding consistency is similar to most church denominations, which would call for a different tag like `operator`. I suppose the same could be said for most of these social centers though.

**Comment by quincylvania** [comment]
@1ec5 True. I still think it'd still be useful to suggest consistent tags for these kind of social centers in iD. I think the main difference between these and most brands is that the name could vary?

That said, I don't care strongly about adding them.

**Comment by 1ec5** [comment]
There’s a lot of brands in this index where the name can vary, so that alone wouldn’t determine whether it gets included. But I guess it comes down to the meaning of a “brand” versus an operator, which is a distinction I’m less and less sure about these days.

I also added several organizations as social center “brands” with the reasoning that they’re tagged very inconsistently. We haven’t even begun adding labor unions, and they all have lots of local halls too. But I think we’re at the very limit of the index’s current scope, and I’m wondering if we should actually remove some social center entries from the index.

There is a need for more consistent tagging of things that clearly don’t warrant `brand` tags, like public transportation facilities and places of worship. Public transportation already has a `network` tag for clarifying affiliation, and places of worship have `denomination`. Even when a denomination has [an abundance of common symbolism](https://wiki.openstreetmap.org/wiki/Tag:denomination%3Dla_luz_del_mundo#How_to_identify) from location to location, the locations don’t get `brand` tags. I do think we should expand the index’s scope at some point, but it needs to be deliberate, because clients are probably using this index in ways that are specific to commercial branding.

**Comment by bhousel** [comment]
I think these are all ok and we should include them... Currently in iD, if the preset includes both `brand` and `name` fields, users are allowed to customize the `name` field.   (Anyone who really knows what they are doing can change the `name` tag by editing the raw tags or using a different editor, we just encourage users to leave the name field alone in most cases when dealing with branded stuff).
